### DISCUSSION TITLE
Draft release notes: v1.16.0-beta

### DISCUSSION THREAD

**Discussion by nbolton** [deskflow/deskflow#7491] [Ideas]
This is a draft and may change.

# Synergy v1.16.0-beta

We would like to announce experimental Wayland support for Synergy on Linux. Wayland is a replacement for the X11 window system protocol and architecture.

Special thanks:
- Peter Hutterer (@whot) and Olivier Fourdan (@ofourdan) from Red Hat for their work on implementing Wayland support. Peter Hutterer's work on [`libei`](https://gitlab.freedesktop.org/libinput/libei) and [`libportal`](https://github.com/flatpak/libportal) is fundamental for Wayland support for Synergy. Thanks to @p12tic for reviewing Peter's work and @GeorgesStavracas for releasing `libportal` 0.8.0 which makes Peter's input capture feature available.
- We would also like to give an honorable mention to Andrew Nelless (@nlyan) and Xinyu Hou (@xhou-wavefin), who worked on the Synergy development team in 2017 and prototyped a Wayland Synergy client using `/dev/uinput` which was demonstrated to Red Hat in the same year.

It is worth noting that Peter initially developed Wayland support in a fork of Synergy called "Barrier", a now dead fork (superseded by the Input Leap fork where Wayland support was further developed). Synergy users who switched to Barrier to get Wayland support can now switch back and use either the free/gratis Synergy 1 Community Edition or the licensed Synergy 1 build.

[ChangeLog]

**Comment by whot** [comment]
> It is worth noting that Wayland support was originally developed by Peter in a fork of Synergy called "Barrier" which is now a dead fork. 

fwwi, most of the development and testing was on the [input-leap](https://github.com/input-leap/input-leap/) code base with reviews from @p12tic. Barrier is only dead for gitlab repository ownership reasons, input-leap is where the active barrier maintainers moved to.

**Reply by nbolton** [reply]
Aha, thanks for clarifying. I've updated the draft. How's that looking now?

**Reply by whot** [reply]
thanks for updating it
